plugins/outputs/influxdb/influxdb.go

```diff
@@ -167,13 +169,17 @@ func (i *InfluxDB) Connect() error {
 				return err
 			}
 
+			c.SetLogger(i.log)
```

Comment:
Just pass it over with the rest of the config?

Reply:
You mean in the call to `i.udpClient(parts, i.log)` and `i.httpClient(ctx, parts, proxy, i.log)`?